test/end-to-end/edit_journal/edit_journal.spec.js

```diff
@@ -0,0 +1,146 @@
+/* global browser, element, by */
+const chai = require('chai');
+const helpers = require('../shared/helpers');
+const FU = require('../shared/FormUtils');
+const GU = require('../shared/GridUtils');
+const components = require('../shared/components');
+
+helpers.configure(chai);
+const expect = chai.expect;
+
+describe('Edit Posting Journal', () => {
+  const path = '#!/journal';
+  const gridId = 'journal-grid';
+  before(() => helpers.navigate(path));
+
+  var accountKey = 11;
+
+  it('edits a transaction change an account', function () {
+    // click the "grouping" button
+    FU.buttons.grouping();
+    element.all(by.css('[class="ui-grid-icon-plus-squared"]')).get(0).click();
+    element.all(by.css('[class="fa fa-edit"]')).get(0).click();
+    const accountNumberCell = GU.getCellName(gridId, 1, 4);
+    // simulate dbl click
+    browser.actions().mouseMove(accountNumberCell).doubleClick().perform();
+    
+    accountNumberCell.element(by.css("input")).sendKeys(protractor.Key.chord(protractor.Key.CONTROL, "a"));
+    accountNumberCell.element(by.css("input")).sendKeys(protractor.Key.BACK_SPACE);
+
+    accountNumberCell.element(by.css("input")).sendKeys(accountKey);
+    element.all(by.css('[title="1100 - Test Capital One"]')).click();
+
+    element.all(by.css('[class="fa fa-save"]')).click();
+
+    components.notification.hasSuccess();
+    element.all(by.css('[class="ui-grid-icon-minus-squared"]')).get(0).click();
+    FU.buttons.grouping();
+  });
+
+
+  it('edits a transaction change value of Debit and Credit', function () {
+    FU.buttons.grouping();
+    element.all(by.css('[class="ui-grid-icon-plus-squared"]')).get(1).click();
+    element.all(by.css('[class="fa fa-edit"]')).get(1).click();
+    
+    const debitCell = GU.getCellName(gridId, 2, 5);
+    const creditCell = GU.getCellName(gridId, 3, 6);
+    // simulate dbl click
+    browser.actions().mouseMove(debitCell).doubleClick().perform();
```

Comment:
This would be easier to read if you put this in a function.  Like this:
```js
// simulates a double click
const doubleClick = (element) => browser.actions().mouseMove(element).doubleClick().perform();

// ...then later...
it('edits a transaction change value of Debit and Credit', function () {
  const debitCell = GU.getCellName(gridId, 2, 5);
  doubleClick(debitCell);
});
```